src/test/packages/38-ephemeral-container/zarf.yaml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/zarf-dev/zarf/v0.49.1/zarf.schema.json
+kind: ZarfPackageConfig
+
+metadata:
+  name: basic-pod
+  version: 0.0.1
+
+components:
+  - name: busybox
+    required: true
+    manifests:
+      - name: busybox
+        namespace: test
+        files:
+          - pod.yaml
+    images:
+      - busybox:1.36
```

Comment:
Unfortunately, github actions runners tend to get rate limited from dockerhub pretty quickly so we tend to keep image pulls to ghcr. Probably something we should fix, but for now I'd recommend using `ghcr.io/stefanprodan/podinfo:6.4.0` or something else from ghcr.